templates/amm.cpp.in

```diff
@@ -314,9 +314,20 @@ double calculate_amm_impl(const @ModelName@_mass_eigenstates& model, const softs
 
 @AMMCalculation@
 
-   if (!is_zero((form_factors[2] + form_factors[3]).imag())) {
-      ERROR("Error in the g-2 calculation! Form factor F2 should be real");
-      return std::numeric_limits<double>::quiet_NaN();
+   // Im[F2]/Re[F2] ~ 0
```

Comment:
I'd be in favor of removing this error handling altogether, if you don't mind.

Reply:
I'm against removing removing this. This is exactly the thing that let us find out that there's an error in the first place.

Reply:
I see your point, it finds a bug on the FS side. I guess the error message "Error in the g-2 calculation! Form factor F2 should be real" is clear enough to express this.

Reply:
Thanks. This is a relatively not costly check. It shouldn't be a major bottleneck.